creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -560,6 +560,8 @@ const handleSyncFromPrimary = async (
 
         await transaction.rollback()
 
+        await DBManager.fixInconsistentUser(fetchedCNodeUser.cnodeUserUUID)
```

Comment:
thought behind running this on every error, instead of checking that its due to the expected error?

also a log statement would be very helpful here so we can track

Reply:
@dmanjunath do you remember why we put the fix in the catch rather than on the specific error? I remember we talked about it

Reply:
i think we can gate it on the non-contiguous error. these are both good ideas. not checking err doesn't hurt us beyond it being a little inefficient

Reply:
in progress here https://linear.app/audius/issue/CON-305/fixinconsistentuser-bugfix-logs